src/components/Calendar/Calendar.tsx

```diff
@@ -71,6 +71,17 @@ const CalendarStyling = styled.div`
     }
 `;
 
+const endDateInclusiveEvents = (endDateExclusiveEvents: CalendarEvent[]): CalendarEvent[] => {
+    return endDateExclusiveEvents.map((exclusiveEvent: CalendarEvent) => {
```

Comment:
arrow function should have a return type on it

Reply:
Return type, has been added.